spec/models/profile_spec.rb

```diff
@@ -85,7 +85,6 @@
     before do
       create(:profile_field, label: "Test 1")
       create(:profile_field, label: "Test 2", input_type: :check_box)
-      described_class.refresh_attributes!
```

Comment:
We added this to the `profile_fields` factor a while ago so a lot of these explicit calls can go away:

```ruby
after :create do
  # this is accomplished by ProfileFields::Add normally, it was added here
  # in case the tests use the factory and not the service object
  Profile.refresh_attributes!
end
```